cityofaustin/atd-moped
# enhances project types activity log events

this pr refines project types activity log events using REST connectors.

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/11310

## Testing
**URL to test:** 

please test locally as this pr is comparing against another activity log branch

**Steps to test:**
- add, edit and remove a project type while noting the activity log updates
- the changes should be reflected in the activity log

---
#### Ship list
